jmx-scraper/src/integrationTest/java/io/opentelemetry/contrib/jmxscraper/JmxScraperContainer.java

```diff
@@ -28,7 +28,7 @@ public class JmxScraperContainer extends GenericContainer<JmxScraperContainer> {
   private final Set<String> customYamlFiles;
 
   public JmxScraperContainer(String otlpEndpoint) {
-    super("openjdk:8u272-jre-slim");
+    super("openjdk:8u342-jre-slim");
```

Comment:
[for reviewer] Updated to most recent version of Java8 and to support arm64 arch